### DISCUSSION TITLE
IKEA TRADFRI signal repeater issues

### DISCUSSION THREAD

**Discussion by arduino43** [Koenkk/zigbee2mqtt#5415] [Devices]
I have two IKEA blinds connected using the IKEA TRADFRI signal repeater, but am running into issues with other devices in the same room connecting to the repeater and becoming unresponsive. 

Does anyone else seem to have this issue? Is there any specific configuration I can set to avoid devices connecting to the repeater? It is my understanding that this is not possible.

**Comment by pfurrie** [comment]
Wow... that's a while ago, no comments, and today I'm having the same problem (two new IKEA blinds connected to TRADFRI repeaters, and two IDEA light bulbs in the same room have now become unresponsive.  I've removed the repeaters (do I really need them?), but the problem persists.

**Reply by arduino43** [reply]
I removed the repeaters and repaired all of my devices - no issues since doing this, but a hassle at the time as I had 50+ devices.

**Comment by hmoffatt** [comment]
My Tradfri repeater keeps dropping off the network and is a pain to re-join. I think I'm going to give up on it.
